src/test/java/edu/ucla/library/services/metadata/MetadataSetterTest.java

```diff
@@ -163,15 +163,12 @@ public void testPopulateMediaFormat() throws Exception {
      */
     @Test
     public void testGetMetaFromBadFileMultipleMediaPaths() throws Exception {
-        final String ffprobeMessage = "/usr/bin/ffprobe returned non-zero exit status." + System.lineSeparator();
+        final String errorMessage = "Problem reading media file";
```

Comment:
Could put this string in the messages file.

Reply:
This was actually a fragment of a message bundle entry (MG-106), but I've updated the test to explicitly build the comparison string from the message bundle.